<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutAppWidget</class>
 <widget class="QWidget" name="AboutAppWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>63</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="dsc_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>421</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="front_addr_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="rear_addr_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="rear_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </widget>
  <widget class="QLabel" name="front_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
